pybaseball/amateur_draft.py

```diff
@@ -0,0 +1,44 @@
+import pandas as pd
+import requests 
+from bs4 import BeautifulSoup
+import sys
+from PyQt5.QtWidgets import QApplication
+from PyQt5.QtCore import QUrl
+from PyQt5.QtWebEngineWidgets import QWebEnginePage
+
+class Client(QWebEnginePage):
```

Comment:
Baseball America's draft results page is rendered dynamically, so I needed to use PyQT5 to mimic a browser. I couldn't find a simpler way to do this; lmk if you know of a better way.